spec/fluent.ebnf

```diff
@@ -3,13 +3,24 @@
 Resource            ::= (blank_line | Entry | junk_line)*
 Entry               ::= Message
                       | Term
-                      | (ResourceComment | GroupComment | Comment)
-Message             ::= Comment? Identifier inline_space? "=" inline_space? ((Pattern Attribute*) | (Attribute+)) line_end
+                      | Section
+                      | ValueLessMessage
```

Comment:
I like `LegacyComment` and I wonder if in a similar vein we could have `LegacySection` and `LegacyMessage` (or `LegacyValueless`) here?

Just a suggestion, the current names LGTM as well, with a nit about `ValueLess`: should it be spelled `Valueless` with a lowercase `L`?